openstudiocore/src/airflow/contam/ForwardTranslator.cpp

```diff
@@ -434,10 +434,18 @@ boost::optional<contam::IndexModel> ForwardTranslator::translateModel(model::Mod
   m_logSink.setThreadId(QThread::currentThread());
   m_logSink.resetStringStream();
 
-  m_prjModel.read(std::string(":/templates/template.prj"));
-  if (!m_prjModel.valid()) {
-    return boost::none;
+  {
+    QFile f(":/templates/template.prj");
```

Comment:
It would be nice to have something to replace embedded resources, although if we can't have that we could use resources on disk.  That makes openstudio.exe less self contained though